src/sensors/bno080sensor.cpp

```diff
@@ -119,6 +120,9 @@ void BNO080Sensor::motionLoop()
             imu.getGameQuat(quaternion.x, quaternion.y, quaternion.z, quaternion.w, calibrationAccuracy);
             quaternion *= sensorOffset;
 
+            uint8_t accelAccuracy = 0;
+            imu.getLinAccel(this->acceleration[0], this->acceleration[1], this->acceleration[2], accelAccuracy);
```

Comment:
check for` imu.hasNewAccel() `in the code.  there gets data read from `imu.getAccel(v[0], v[1], v[2], acc);` and send over the network. I think this is done here twice.

Reply:
Yup it is being done twice, should  probably change a few other things I realized could lead to problems as well